<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SnesColorPicker</class>
 <widget class="QDialog" name="SnesColorPicker">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>205</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>205</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>205</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Change color...</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="vLayMain">
   <property name="spacing">
    <number>10</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="hLayTabsPreview">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tabRGB">
        <attribute name="title">
         <string>RGB</string>
        </attribute>
        <layout class="QVBoxLayout" name="vLayRgb">
         <item>
          <layout class="QHBoxLayout" name="hLayRed" stretch="0,8,0">
           <item>
            <widget class="QLabel" name="lblRed">
             <property name="text">
              <string>R</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="slRed">
             <property name="maximum">
              <number>31</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txtRed">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>42</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="baseSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="inputMask">
              <string/>
             </property>
             <property name="maxLength">
              <number>2</number>
             </property>
             <property name="frame">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="hLayGreen" stretch="0,8,0">
           <item>
            <widget class="QLabel" name="lblGreen">
             <property name="text">
              <string>G</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="slGreen">
             <property name="maximum">
              <number>31</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txtGreen">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>42</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="maxLength">
              <number>2</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="hLayBlue" stretch="0,8,0">
           <item>
            <widget class="QLabel" name="lblBlue">
             <property name="text">
              <string>B</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="slBlue">
             <property name="maximum">
              <number>31</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txtBlue">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>42</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="maxLength">
              <number>2</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabHsv">
        <attribute name="title">
         <string>HSV</string>
        </attribute>
        <layout class="QVBoxLayout" name="vLayHsv">
         <item>
          <layout class="QHBoxLayout" name="hLayHue" stretch="0,8,0">
           <item>
            <widget class="QLabel" name="lblHue">
             <property name="text">
              <string>H</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="slHue">
             <property name="maximum">
              <number>360</number>
             </property>
             <property name="pageStep">
              <number>20</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txtHue">
             <property name="maximumSize">
              <size>
               <width>42</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="maxLength">
              <number>3</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="hLaySat" stretch="0,8,0">
           <item>
            <widget class="QLabel" name="lblSat">
             <property name="text">
              <string>S</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="slSat">
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="singleStep">
              <number>1</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txtSat">
             <property name="maximumSize">
              <size>
               <width>42</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="inputMask">
              <string/>
             </property>
             <property name="maxLength">
              <number>3</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="hLayValue" stretch="0,8,0">
           <item>
            <widget class="QLabel" name="lblValue">
             <property name="text">
              <string>V</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="slValue">
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txtValue">
             <property name="maximumSize">
              <size>
               <width>42</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="inputMask">
              <string/>
             </property>
             <property name="maxLength">
              <number>3</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="ColorPreview" name="wColorPrev" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hLayButtonBox">
     <property name="spacing">
      <number>8</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorPreview</class>
   <extends>QWidget</extends>
   <header>colorpreview.moc.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
